build/commands/lib/config.js

```diff
@@ -262,6 +262,8 @@ const Config = function () {
     'bitflyer_sandbox_client_secret',
     'bitflyer_sandbox_fee_address',
     'bitflyer_sandbox_url',
+    'brave_account_api_endpoint',
```

Comment:
we are not adding new endpoints like this anymore, use `brave_domain::GetServicesDomain`